pysoa/common/transport/redis_gateway/backend/base.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, redis_connection):  # type: (redis.StrictRedis) -> None
 
         :param redis_connection: The connection for registering this script
         """
-        self._redis_script = redis_connection.register_script(self._script.strip())  # type: redis.client.Script
+        self._redis_script = redis_connection.register_script(self._script.strip())  # type: ignore
```

Comment:
This change was due to the fact that for some reason mypy couldn't find the `redis.client.Script` type and was throwing an error on it.

Reply:
I think what happened is the type annotation was probably added while mypy wasn't actually being run due to travis CI being down (or someone just merged with a breaking build since they don't seem to be blocking PRs). All tests still pass and as mypy shouldn't affect runtime I think we should be ok. Is there another level of testing you had in mind for us to run on this?

Reply:
I would say this must be tested in QA at least. Bumping the new version on your service, then build, then deploy.

Reply:
Definitely will do! This is going into soa_proxy_service so I will work with Luca and others to make sure that everything is verified there.